proposals/wac-ucr/index.bs

```diff
@@ -802,6 +802,29 @@ For example, if the data Carol and Oscar saw in the resume was
 background, she wouldn't want them to know that they were only seeing
 a filtered view.
 
+### Minimal Credential Disclosure ### {#uc-minimalcredentials1}
+
+To continue with the  [[#uc-whopermitted]] example, Oscar now wants to
+view Alice's `resume` which is not publically accessible. To gain access,
+he needs to authenticate with the right credentials. Oscar does not want
+to try out each one of his credentials one by one until he gains access,
+as that would both be slow and allow Alice to connect his different personas.
+
+### Minimal Credentials Disclosure in an Open World
+
+Alice enjoys using her Solid Blog editor (BlogEd) that automatically discovers the
+blogs of all her family, friends and colleagues as well as allow her to follow 
+the blogs she finds on the internet.
+
+BlogEd allows her to add comments on any post she finds. Most
+blogs have restrictions on who can post to avoid spam. Alice has direct
+access to many of them, but they don't all know her under the same
+identitfier. Some provide access via social network relationships, such
+as being the friend of a friend of the Blog's author. Alice would like to
```

Comment:
If you want to focus on FoF based scenarios, I would suggest to focus this Use case on that and clarify details around how storage server can or can not access information around friendships.

Reply:
I wrote up a use case for giving comment access to extended social network that should help with reworking this issue. https://github.com/solid/authorization-panel/pull/98/commits/a4eb4ac3afda66b0bf726b06be00827ca85d524a

Reply:
Agree - this one feels more specific to a FoF than related to minimal credential disclosure. I think this should focus on the FoF case since the minimal disclosure is covered well in uc-minimalcredentials